HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -126,33 +127,84 @@ private void saveMap()
 
         private void Edit_Display_Information(object sender, RoutedEventArgs e)
         {
+            this.ObjectTypeChooserComboBox.SelectedItem = DisplayMapObject.MapObjectType.ObjectTypeFullName;
             EditMode = !EditMode;
```

Comment:
I'm actually having a bug with this - when I enter edit mode, the object type isn't selected in the combo box:
![image](https://user-images.githubusercontent.com/57844740/99149730-2d2cf800-2690-11eb-9ca1-734268aeda9b.png)
I'm not sure why this is happening since this bit of code should be setting it, so maybe look into that.

Reply:
i dont think that can happen, because if you enter edit mode and no object is selected edit mode is disabled

Reply:
can you please take a screenshot of that situation, l check it several times and it never happened

Reply:
I will take care of it.

Reply:
thank you, l fix that bug..if you do not choose any mapObject type edited object  will have type of selected object.